### PR TITLE
chore(stable-env): change config management

### PR DESCRIPTION
**Pull request by evegufy**
## Description

app setup in ArgoCD: retire values-stable.yaml and replace it with config from HELM_VALUES env var

## Why

Deployment mode change

## Issue

n/a

## Checklist

- [x] I have successfully tested my changes

### CODE REVIEW

File: consortia/argocd-app-templates/appsetup-stable.yaml [hunk 1]
```diff
@@ -23,19 +23,230 @@ metadata:
...
+                custodian:
+                  clientId: "<path:portal/data/processes-worker#custodian-client-id>"
+                  clientSecret: "<path:portal/data/stable/processes-worker#custodian-client-secret>"
+                sdfactory:
+                  issuerBpn: "BPNL00000003CRHK"
+                  clientId: "<path:portal/data/processes-worker#sdfactory-client-id>"
+                  clientSecret: "<path:portal/data/stable/processes-worker#sdfactory-client-secret>"
+                offerprovider:
+                  clientId: "<path:portal/data/processes-worker#offerprovider-client-id>"
+                  clientSecret: "<path:portal/data/stable/processes-worker#offerprovider-client-secret>"
```
Comment (Phil91) [inline consortia/argocd-app-templates/appsetup-stable.yaml:237]: there was a new secret introduced for onboarding-service-provider

**Review by Phil91 — CHANGES_REQUESTED** [review]

File: consortia/argocd-app-templates/appsetup-stable.yaml (hunk 1 above)
Comment (evegufy) [reply consortia/argocd-app-templates/appsetup-stable.yaml:239]: thanks, I added https://github.com/eclipse-tractusx/portal-cd/pull/91/files#diff-655f983f01cd391de5332255653b7b05cb3be87d3f9a6876ec90276e494b6287R163

File: consortia/argocd-app-templates/appsetup-stable.yaml [hunk 2]
```diff
@@ -23,19 +23,232 @@ metadata:
...
+                        backend:
+                          service: "marketplace-app-service"
+                          port: 8080
+                      - path: "/api/services"
+                        pathType: "Prefix"
+                        backend:
+                          service: "services-service"
+                          port: 8080
+              keycloak:
+                central:
```
Comment (Phil91) [inline consortia/argocd-app-templates/appsetup-stable.yaml:121]: do we need to add the useAuthTrail here and in line 128?

Comment (evegufy) [reply consortia/argocd-app-templates/appsetup-stable.yaml:121]: good point! still for now I'd just leave it in the default values file, I may move it into the env specific config as part of https://github.com/eclipse-tractusx/portal-cd/pull/84

File: consortia/argocd-app-templates/appsetup-stable.yaml (hunk 1 above)
Comment (evegufy) [reply consortia/argocd-app-templates/appsetup-stable.yaml:239]: I did forget [this](https://github.com/eclipse-tractusx/portal-cd/pull/91/commits/9084da642d3437f4ce916a0934bebb162b244eab#diff-655f983f01cd391de5332255653b7b05cb3be87d3f9a6876ec90276e494b6287R240) earlier, should be fine now

**Review by Phil91 — APPROVED** [review]

**Pull request merged**